src/template/template.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from "@nestjs/common";
+import { readFileSync } from "fs";
+import { TemplateContent } from "./dto/template-content.dto";
+import * as Handlebars from "handlebars";
+import mjml2html = require('mjml');
+
+@Injectable()
+export class TemplateService {
+  public getResetPasswordEmailTemplate(locale: string): TemplateContent {
+    let content = new TemplateContent();
+    const MJML_TEMPLATE_PATH = `${__dirname}/templates/${locale}/reset-password.mjml`;
+    content.mjml = readFileSync(MJML_TEMPLATE_PATH, "utf8");
+    return content;
+  }
+  public injectData<T>(content: TemplateContent, data: T): string {
```

Comment:
This method should handle only html content, it shouldn’t be responsible for mjml to html conversion.